sources/packages/backend/libs/integrations/src/esdc-integration/sin-validation/sin-validation.module.ts

```diff
@@ -1,24 +1,26 @@
 import { Module } from "@nestjs/common";
-import { AuthModule } from "../../auth/auth.module";
-import { SINValidationService, StudentService } from "../../services";
 import { SequenceControlService } from "@sims/services";
-import { SFASIndividualService } from "@sims/services/sfas";
 import { SINValidationIntegrationService } from "./sin-validation-integration.service";
 import { SINValidationProcessingService } from "./sin-validation-processing.service";
 import { ConfigModule } from "@sims/utilities/config";
 import { SshService } from "@sims/integrations/services";
+import { StudentService } from "@sims/services/student/student.service";
+import { SINValidationService } from "@sims/services/sin-validation/sin-validation.service";
 
 @Module({
-  imports: [AuthModule, ConfigModule],
+  imports: [ConfigModule],
   providers: [
     SshService,
-    SFASIndividualService,
-    StudentService,
     SINValidationService,
+    StudentService,
     SINValidationIntegrationService,
     SINValidationProcessingService,
     SequenceControlService,
   ],
-  exports: [SINValidationIntegrationService, SINValidationProcessingService],
+  exports: [
+    SINValidationIntegrationService,
```

Comment:
Why `SINValidationIntegrationService` and `SINValidationService` are exported?

They are not used in the scheduler. Am I missing something?

Reply:
mm,, what you mean by not used, 
![image](https://user-images.githubusercontent.com/77353155/210419394-4cd8c827-738d-4c0d-9b90-7561a886077d.png)
![image](https://user-images.githubusercontent.com/77353155/210419563-4cf35a79-272e-411b-b46f-3ced268f345b.png)